crates/boundless-cli/src/bin/boundless-cli.rs

```diff
@@ -109,6 +110,17 @@ enum Command {
 
     /// Display configuration and environment variables
     Config {},
+
+    /// Print shell completions (e.g. for bash or zsh) to stdout.
+    ///
+    /// Bash:
+    ///
+    /// boundless-cli completions bash >> ~/.local/share/bash-completion/completions/boundless-cli
```

Comment:
Would be better to overwrite as we do with zsh?
```suggestion
    /// boundless-cli completions bash > ~/.local/share/bash-completion/completions/boundless-cli
```

Reply:
I decided to just remove these instructions, instead of giving bad advice